scikeras/wrappers.py

```diff
@@ -63,7 +63,7 @@ class or if passed as part of sk_params,
     2. An instance of a class that implements the `__call__` method
     3. An instance of a Keras Model. A copy of this instance will be made
     4. None. This means you implement a class that inherits from `BaseWrapper`,
-    `KerasClassifier` or `KerasRegressor`. The `__call__` method of the
+    `KerasClassifier` or `KerasRegressor`. The `_keras_build_fn` method of the
```

Comment:
Could you add a line documenting the parameters passed to `_keras_build_fn`?

Reply:
They're dynamically determined from the signature of `_keras_build_fn`. I'll add a note. And I still need to update the docs for this change/PR.